site/blog/tutorials/iasql-on-ec2.mdx

```diff
@@ -0,0 +1,260 @@
+---
+slug: iasql-on-ec2
+title: Deploy IaSQL on EC2 using IaSQL
+authors: [mtp1376]
+---
+
+We're intending to publish a series of blog posts on deploying different services using IaSQL. IaSQL covers a wide range of AWS services, so there are lots of possibilities to explore. You can deploy containers to ECS, launch servers using EC2, and have CodePipelines that take the burden of building and deploying your applications.
```

Comment:
CodePipelines as it is is not a name used by aws... maybe "and use AWS Codepipeline to take the burden..."